### PR TITLE
Update GH pages and other things

### PR DESCRIPTION
**Pull request by paweljakubas**
https://github.com/IntersectMBO/cardano-addresses/issues/232

### CODE REVIEW

**Review by dermetfan — APPROVED** [review]
LGTM

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 If you find a bug or you'd like to propose a feature, please feel free to raise
-an issue on our [issue tracker](https://github.com/input-output-hk/cardano-addresses/issues).
+an issue on our [issue tracker](https://github.com/IntersectMBO/cardano-addresses/issues).
 
 Pull requests are welcome!
 
 When creating a pull request, please make sure that your code adheres to our
-[coding standards](https://github.com/input-output-hk/adrestia/blob/master/docs/code/Coding-Standards.md).
+[coding standards](https://github.com/IntersectMBO/adrestia/blob/master/docs/code/Coding-Standards.md).
```
Comment (Anviking) [inline CONTRIBUTING.md:7]: Doesn't work — probably the old link should be kept or the file should be copied to this repo?

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -1,3 +1,3 @@
 See [code of conduct].
 
-[code of conduct]: https://github.com/input-output-hk/cardano-engineering-handbook/blob/main/CODE-OF-CONDUCT.md
+[code of conduct]: https://github.com/IntersectMBO/cardano-engineering-handbook/blob/main/CODE-OF-CONDUCT.md
```
Comment (Anviking) [inline CODE_OF_CONDUCT.md:3]: Doesn't work

File: ChangeLog.md [hunk 3]
```diff
@@ -34,7 +34,7 @@
 ### Added
 
 - Adds the `cardano-address key walletid` command, which derives a
-  [`cardano-wallet` wallet ID](https://input-output-hk.github.io/cardano-wallet/design/WalletId)
+  [`cardano-wallet` wallet ID](https://IntersectMBO.github.io/cardano-wallet/design/WalletId)
```
Comment (Anviking) [inline ChangeLog.md:37]: Doesn't work

**Review by Anviking — COMMENTED** [review]
Noticed a few broken links

File: CONTRIBUTING.md (hunk 1 above)
Comment (paweljakubas) [reply CONTRIBUTING.md:7]: thanks! Fixed

File: CODE_OF_CONDUCT.md (hunk 2 above)
Comment (paweljakubas) [reply CODE_OF_CONDUCT.md:3]: Thanks! Fixed

File: ChangeLog.md (hunk 3 above)
Comment (paweljakubas) [reply ChangeLog.md:37]: Thanks! Fixed

File: ChangeLog.md [hunk 4]
```diff
@@ -34,7 +34,7 @@
 ### Added
 
 - Adds the `cardano-address key walletid` command, which derives a
-  [`cardano-wallet` wallet ID](https://IntersectMBO.github.io/cardano-wallet/design/WalletId)
+  [`cardano-wallet` wallet ID](https://cardano-foundation.github.io/cardano-wallet/design/WalletId)
```
Comment (Anviking) [inline ChangeLog.md:37]: Hm... still doesn't seem to work but not sure where this should point to 🤷‍♂️

**Review by Anviking — APPROVED** [review]

File: ChangeLog.md (hunk 4 above)
Comment (paweljakubas) [reply ChangeLog.md:37]: probably should be this https://input-output-hk.github.io/adrestia/cardano-wallet/design/WalletId

Comment (paweljakubas) [reply ChangeLog.md]: changed

**Pull request merged**